### DISCUSSION TITLE
Is the use of SVG essential? Can other image formats be used?

### DISCUSSION THREAD

**Discussion by Relequestual** [cncf/landscape2#650] [Q&A]
Hello.
We are in the process of building our own landscape (thanks for the hard work on this).
We have many open source tools that don't have a logo, and many which may have a logo but not in SVG format.
Will it work using different formats?
I haven't got this running locally to test it.

**Comment by tegioz** [comment]
Hi @Relequestual

At the moment it is a requirement as we assume logos are in SVG format in the codebase in a few places (and perform some operations on them based on that fact).

However, we could consider lifting this requirement and leave it up to each landscape maintainers to decide whether they want to use SVG logos or not.

**Reply by tegioz** [reply]
I've just created an issue to track this: https://github.com/cncf/landscape2/issues/651 🙂

**Reply by tegioz** [reply]
We’ve addressed this in #667.
